unic/char/range/src/range.rs

```diff
@@ -1,39 +1,34 @@
 use std::char;
 use std::collections::Bound;
-use std::ops::Range;
-use step;
-
-const SURROGATE_RANGE: Range<u32> = 0xD800..0xE000;
+use CharIter;
 
 /// A range of unicode code points.
 ///
-/// The members of this struct are public for const initialization by `chars!(..=)` only.
-/// They should be considered unstable private API that may change at any time.
-/// If you decide to use them anyway, make sure to note the safety notes.
+/// The most idiomatic way to construct this range is through the use of the `chars!` macro:
+///
+/// ```
+/// # #[macro_use] extern crate unic_char_range;
+/// # use unic_char_range::*;
```

Comment:
It's better to not hide the `extern crate` and `use` lines from the doc code blocks. (One empty line between them, plus an empty line before the actual code.)